### ISSUE TITLE
Some questions about the output

### ISSUE THREAD

**Issue by zhibindaxia** [bobzhuyb/ns3-rdma#26]
Can you tell me where the content in the mix.tr file comes from?What is the meaning of qFb in the figure below?Thank you.
![image](https://user-images.githubusercontent.com/37073118/49873071-e6a44280-fe55-11e8-8602-76fa135d0448.png)

**Comment by bobzhuyb** [comment]
It is generated from here:

https://github.com/bobzhuyb/ns3-rdma/blob/e8a27d02bfe5ef0aec189c7d1a401547dbd8fbad/src/point-to-point/model/cn-header.cc#L123

If I remember correctly, it's for counting how many ECN marks the receiver has received in the last time period. This is not used for now (I experimented with it but later did not really use it)

**Comment by zhibindaxia** [comment]
If qFB is output, can I think there is congestion?

**Comment by bobzhuyb** [comment]
yes

**Comment by zhibindaxia** [comment]
Hi,I read the papers Congestion Control for Large - scale RDMA Deployments, SIGCOMM '15 (DCQCN)  , what would you like to use tool to draw the figure?
![image](https://user-images.githubusercontent.com/37073118/50042720-2bbcb480-00a2-11e9-8a9d-9b83d940c668.png)

**Comment by zhibindaxia** [comment]
Can I know which line of code calls these two functions?
![image](https://user-images.githubusercontent.com/37073118/50150257-54d68280-02f8-11e9-947c-6582500f5a2d.png)
![image](https://user-images.githubusercontent.com/37073118/50150276-63249e80-02f8-11e9-8ede-ebc58cd06074.png)

**Comment by bobzhuyb** [comment]
I used gnuplot for those figures.

The function is called by one of the trace functions, probably in qbb-net-device.cc, but I don't remember which function it is.

**Comment by zhibindaxia** [comment]
Thank you.

**Comment by zhibindaxia** [comment]
Sorry to bother you again, but did you generate these images by calling GnuplotHelper code in visual studio2015?I was unable to call GnuplotHelper in visual studio2015.
![image](https://user-images.githubusercontent.com/37073118/50412543-0f0f4200-0843-11e9-985d-7a87c6b6ca4c.png)

**Comment by bobzhuyb** [comment]
No. I parsed the trace output and plotted the figures using native gnuplot.

**Comment by zhibindaxia** [comment]
Did you get a graph of the throughput of the nodes from the mix.tr file?I'm a little confused about that.

**Comment by bobzhuyb** [comment]
Yes. You need to write some simple scripts yourself.

**Comment by zhibindaxia** [comment]
If it is convenient for you, could you give me a simple example for my reference?Thank you.

**Comment by zhibindaxia** [comment]
![image](https://user-images.githubusercontent.com/37073118/50468534-5af8ed00-09e3-11e9-999b-4df157365bdb.png)
Can you tell me what these values mean and how they relate to each other?

**Comment by zhibindaxia** [comment]
![image](https://user-images.githubusercontent.com/37073118/50944444-50eaea00-14cb-11e9-98cc-b3f8d9e7b75c.png)
Could you tell me the function of this function？
